packages/arb-bridge-peripherals/contracts/tokenbridge/libraries/DecimalConverter.sol

```diff
@@ -4,13 +4,13 @@
 pragma solidity ^0.6.0;
 
 library DecimalConverter {
-  function from777to20(uint256 amount, uint8 decimals) internal pure returns (uint256) {
-    require(decimals <= 18, 'DEC');
-    return amount / (10 ** uint256(18 - decimals));
-  }
+    function from777to20(uint8 decimals, uint256 amount) internal pure returns (uint256) {
```

Comment:
Seems like a regression. The contracts using this library assume the amount is the first parameter.

Reply:
ah good catch (rebase issue: linter made some changes, but didn't notice the changes beyond the lint)